backend/api/api_deployment_views.py

```diff
@@ -119,9 +125,27 @@ def create(
             headers=headers,
         )
 
-
-def get_error_from_serializer(error_details: dict[str, Any]) -> Optional[str]:
-    error_key = next(iter(error_details))
-    # Get the first error message
-    error_message: str = f"{error_details[error_key][0]} : {error_key}"
-    return error_message
```

Comment:
@chandrasekharan-zipstack why was this removed? @muhammad-ali-e is it safe do this?

Reply:
@ritwik-g we were previously using this to handle errors from serializers. This was incorrect and anyway we have replaced it with [drf-standardized-error](https://drf-standardized-errors.readthedocs.io/en/latest/index.html). So what I've removed is dead code - it wasn't being used anywhere either